Sources/Profile-View.php

```diff
@@ -324,11 +324,20 @@ function fetch_alerts($memID, $all = false, $counter = 0, $pagination = array())
 		if (!empty($alert['text']))
 			continue;
 		if (isset($alert['extra']['board']))
-			$alerts[$id_alert]['extra']['board_msg'] = $boards[$alert['extra']['board']];
+			if ($boards[$alert['extra']['board']] == $txt['board_na'])
+				unset($alerts[$id_alert]);
+			else
+				$alerts[$id_alert]['extra']['board_msg'] = $boards[$alert['extra']['board']];
```

Comment:
I'm only looking at the code here and not the context, but why are we checking this against $txt?

Reply:
all alerts where intialize with the txt https://github.com/albertlast/SMF2.1/blob/82522334aa8ee193ba505ed2a8c354be1ce8259e/Sources/Profile-View.php#L263
and i placed the check here if the intial text is in here or changed.
When the array got the intial value, the user doesn't hav the access right -> remove this from the list.

Reply:
Ok, seems weird we are doing the N/A, but ok.